src/fixed-footer-layout.tsx

```diff
@@ -126,23 +126,23 @@ const FixedFooterLayout: React.FC<Props> = ({
             })}
             // this style is inline to avoid creating a class  that may collide with
             // other fixed footers during the page animation transition
-            style={{background: isMobile ? footerBgColor || colors.background : undefined}}
+            style={{background: isTabletOrSmaller ? footerBgColor || colors.background : undefined}}
             data-testid={`fixed-footer${isFooterVisible ? '-visible' : '-hidden'}`}
         >
             {isFooterVisible && (
-                <div style={{height: footerHeight, marginBottom: 'env(safe-area-inset-bottom)'}}>
+                <aside style={{height: footerHeight, marginBottom: 'env(safe-area-inset-bottom)'}}>
```

Comment:
Isn't `<footer>` better? what's the a11y report error we had with `div`?

Reply:
a footer has different semantics:
> A `<footer>` typically contains information about the author of the section, copyright data or links to related documents.

Reply:
you can inspect previous a11y issues here:
![image](https://user-images.githubusercontent.com/579417/129183487-cf126cba-2881-4d1b-91f6-439f3ea778b1.png)

Example report:
https://devmisticaweb.blob.core.windows.net/mistica-web-1628677668961/components-layouts-fixedfooterlayout--dialog-over-fixed-footer.html